README.md

```diff
@@ -120,6 +120,16 @@ Files which match will not be formatted. Set to `null` to not read ignore files.
 A list of languages IDs to disable this extension on. Restart required.
 *Note: Disabling a language enabled in a parent folder will prevent formatting instead of letting any other formatter to run*
 
+#### prettier.jsxSingleQuote (default: false)
+Use single quotes instead of double quotes in JSX.
+
+#### prettier.htmlWhitespaceSensitivity (default: 'css')
+Specify the global whitespace sensitivity for HTML files. [Learn more here](https://prettier.io/docs/en/options.html#html-whitespace-sensitivity)
+
+
+#### prettier.htmlWhitespaceSensitivity (default: 'auto')
+Specify the end of line used by prettier. [Learn more here](https://prettier.io/docs/en/options.html#end-of-line)
+
```

Comment:
Settings shoulb be put in prettier's settings, not vscode settings for prettier.
After `#### prettier.arrowParens (default: 'avoid')`

Reply:
Done.